src/zcash/Address.cpp

```diff
@@ -103,15 +107,14 @@ std::vector<const Receiver*> UnifiedAddress::GetSorted() const {
     return sorted;
 }
 
-bool UnifiedAddress::AddReceiver(Receiver receiver) {
+bool UnifiedAddress::AddReceiver(Receiver receiver)
+{
     auto typecode = std::visit(TypecodeForReceiver(), receiver);
     for (const auto& r : receivers) {
         auto t = std::visit(TypecodeForReceiver(), r);
-        if (
-            (t == typecode) ||
-            (std::holds_alternative<CKeyID>(r) && std::holds_alternative<CScriptID>(receiver)) ||
-            (std::holds_alternative<CScriptID>(r) && std::holds_alternative<CKeyID>(receiver))
-        ) {
+        if ((t == typecode)
+            || (std::holds_alternative<CKeyID>(r) && std::holds_alternative<CScriptID>(receiver))
+            || (std::holds_alternative<CScriptID>(r) && std::holds_alternative<CKeyID>(receiver))) {
             return false;
         }
```

Comment:
This would become
```c++
        if ((t == typecode)
            || (std::holds_alternative<CKeyID>(r) && std::holds_alternative<CScriptID>(receiver))
            || (std::holds_alternative<CScriptID>(r) && std::holds_alternative<CKeyID>(receiver)))
        {
            return false;
        }
```
with [AfterControlStatement: MultiLine](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#bracewrapping) as suggested [here](https://github.com/zcash/zcash/pull/6678/files#r1200474146).